pandas/tests/io/json/test_normalize.py

```diff
@@ -634,6 +634,23 @@ def test_missing_meta(self, missing_metadata):
         expected = DataFrame(ex_data, columns=columns)
         tm.assert_frame_equal(result, expected)
 
+    def test_missing_nested_meta(self):
+        # GH44312
+        # If errors="ignore" and metadata is null, we should it even if it's nested
+        data = {"meta": "foo", "nested_meta": None, "value": [{"rec": 1}, {"rec": 2}]}
+        result = json_normalize(
+            data,
+            record_path="value",
+            meta=["meta", ["nested_meta", "leaf"]],
+            errors="ignore",
+        )
+        ex_data = [["foo", np.nan, 1], ["foo", np.nan, 2]]
+        columns = ["meta", "nested_meta.leaf", "rec"]
+        expected = DataFrame(ex_data, columns=columns).astype(
+            {"nested_meta.leaf": object}
+        )
+        tm.assert_frame_equal(result, expected, check_like=True)
```

Comment:
why check_like? is there an ordering issue?

Reply:
laziness. fixed.